### DISCUSSION TITLE
Update to 2.2.12 stable from 1.75 stable

### DISCUSSION THREAD

**Discussion by 7djs7** [Nico640/docker-unms#89] [Q&A]
Hi, I'm inquiring to see when the new stable release will be ready? It was just recently released and i'm curious how long it takes to update from each stable release. Thanks!

**Comment by Nico640** [comment]
Hi,

between minor releases it normally only takes like one day, but because it's a rather big update and I haven't had a lot of spare time to work on it lately, it has taken a bit longer. But it's updated on the testing branch now and it will be on master soon :)
